### PR TITLE
Update expired certificates

### PR DESCRIPTION
**Pull request by bbondy**
### Summary

When I ran the grants container I got these errors:
> DBG failed to dial: x509: certificate has expired or is not yet valid: current time 2024-10-29T18:07:57Z is after 2024-04-02T16:18:51Z module=kafka.NewKafkaReader
> DBG Unable to establish connection to consumer group coordinator for group grant-bat-skus-local: failed to dial: x509: certificate has expired or is not yet valid: current time 2024-10-29T18:08:02Z is after 2024-04-02T16:18:51Z module=kafka.NewKafkaReader
> DBG failed to dial: x509: certificate has expired or is not yet valid: current time 2024-10-29T18:08:02Z is after 2024-04-02T16:18:51Z module=kafka.NewKafkaReader

I verified that the certificates were expired:
```
$ openssl x509 -in test/secrets/snakeoil-ca-1.crt -noout -text | grep 'Not'
            Not Before: Apr  3 16:18:51 2023 GMT
            Not After : Apr  2 16:18:51 2024 GMT
```

To regenerate the certificates I used `test/secrets/README.md`

### Type of Change

- [ ] Product feature
- [ ] Bug fix
- [ ] Performance improvement
- [ ] Refactor
- [x] Other

### Tested Environments

- [x] Development
- [ ] Staging
- [ ] Production

### Before Requesting Review

- [x] Does your code build cleanly without any errors or warnings?
- [x] Have you squashed all intermediate commits?
- [x] Is there a clear title that explains what the PR does?
- [x] Have you performed a self review of this PR?

Not applicable:
- [ ] Have you used auto closing keywords?
- [ ] Have you added tests for new functionality?
- [ ] Have validated query efficiency for new database queries?
- [ ] Have documented new functionality in README or in comments?
- [ ] Have you used intuitive function, variable and other naming?
- [ ] Have you requested security and/or privacy review if needed

### CODE REVIEW

File: test/secrets/consumer.client.key [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-4TKJxO3tMGsAs4HmCXaL4hi1eBCfSNff7V78gn5oHloorTRrwW08pRG0mnixBLgn
-ChT7WW+TRQmWf9hFBEUCQQD4mz///8N6LbCmyBnjPkZe0Aw4CDWEqAVA1PA0AyvI
-hIZ0AZpOCOsqSMzkA6l+DVH5MnqrnMBuvHn7omH3keIvAkEA82Eagr0arC4zIQgu
-TUt6z0RBBXkxO01F/Alfrurez0ZVxgAbzFgmlJKzFmxCPvY6o0aMCYKEz3BH0hpb
-nipHNwJAIGU6pcVSV610g4c3uGD5EonRzctkDfFHHfehhWWUYwDB9IPw85I98rih
-2mSPxZk+wVZOneZDPSraSmwSkAw2iQJAXDs8VLddWErZzXbQNTKZg9qV2YEoCxUp
-pG3Y+OzBK0O4VaTM5A+z7zH49FJqeHnPlvixMRlpQ0OFtDWQ6suS2QJAdZm6k1cV
-rO3D7HybiU+2H13q7lRc9CUgy9fHMyowU+Ab5nONiZQat4e4G9mG3vC3hCqGl7uK
-xtPMeens4rlu1Q==
+MIICdgIBADANBgkqhkiG9w0BAQEFAASCAmAwggJcAgEAAoGBAKhWPvrwCl+o74i2
```
[the remark this answers is not in the document]
Comment (bbondy) [reply test/secrets/consumer.client.key:2]: test keys and already in the repo and using a script to be generated

File: test/secrets/snakeoil-ca-1.key [hunk 2]
```diff
@@ -1,30 +1,30 @@
...
-uf2g6xio9FdW2+4skyVTYkNdlzGPJM/IuRUUCNCyItDzxSJ5sLgBexLsPPM2Mi3A
-fW8Z9XmDJKwgQU49Vl0mLmkSBZbDmWzMJ4J67dkuVT7QJFJ6RhqQgmL62pcW/oN6
-E5Dhbq3dy+jSjgHdEea5gq+QSESyho4JSYmdVf1KqQ9/S4+Y8CATo8Sot37m3C9e
-KjkGcIXDqQbnkvvnOkainZxQAOyxi/8jwojvutvXIrA8hm8OW/lLcTxy0hPBtFUT
-Gnt1X3UaS/8ivaIg9IAgwHKmsXamhJqpa6HqvmHD8kaitSCwVVgoLT0fkk+sx5Ry
-wD/y3CFlVb86DZhQuufcHoRNeRV+Bcz+8CMmEy1cFZmYOZw9qdysu5cKBjSrkkPB
-18LOCPjsAUI7g10mCw9n/+nqOXPAYKy7HhDWRxfH2YKmfBln77MEalwtNS9o2NOi
-ymoOVH0kfhRmRNzS06zWHS+jYzlgVVod3zZ6VfmzFm2D/mkqjkTRtOiUkkoBzgLc
-dC0iOYfxJXHSpPlK/fTFCQ==
+MIIFJDBWBgkqhkiG9w0BBQ0wSTAxBgkqhkiG9w0BBQwwJAQQaYTaCPcySarxocpD
```
[the remark this answers is not in the document]
Comment (bbondy) [reply test/secrets/snakeoil-ca-1.key:2]: test keys and already in the repo and using a script to be generated

**Comment by kdenhartog** [comment]
These certs are for testing, not concerned.

**Review by kdenhartog — APPROVED** [review]

**Review by clD11 — APPROVED** [review]
LGTM approved.

**Pull request merged**